### DISCUSSION TITLE
Configure depth of tree inset

### DISCUSSION THREAD

**Discussion by blaize** [fyne-io/fyne#3924] [Q&A]
Hello,
I'm using a tree to organise my data.
I'm actually redesign the ui of my app to be usable on smartphone.

When I open the tree widget, it add a space before the branches name. Each time I open a branch, it add more space. At the end, when I'm at the leaf, I have lost 50% of the smartphone's screen space with just blank, and I can't even see the entire text.

Is there any way to configure the tree to stack the branches and leaf without spacing between one and another ?

Thanks

**Comment by andydotxyz** [comment]
It is currently specified as the inline icon size so the padding gutter aligns going down the scroll.
I guess this could start a feature request if its needed for many usage?

**Reply by blaize** [reply]
Thank for your answer,

I don't know for the other people but I think it could be handy to adjust the size of the padding to optimize the space on the UI.

I'm not good enough at coding to make the patch but I guess it wouldn't be very difficult if the size is already fixed in a constant or something like that.
